basiclti/basiclti-portlet/src/webapp/WEB-INF/sakai/IMSBLTIPortlet.xml

```diff
@@ -1457,8 +1457,8 @@
           <configuration name="final.releaseemail" value="true"/>
           <configuration name="final.custom" value="true"/>
           <configuration name="final.allowsettings" value="true"/>
-          <configuration name="final.allowroster" value="true"/>
-          <configuration name="final.allowoutcomes" value="true"/>
+          <configuration name="final.allowroster" />
+          <configuration name="final.allowoutcomes" value="false"/>
           <configuration name="final.allowlori" value="true"/>
```

Comment:
Oh, I think I got confused because there are three different syntaxes used. It looks like you can either use `false`, `off` or just remove the value entirely. `nyu.law.panopto` uses `false` so that's probably how I ended up at `final`